cp-ksqldb-cli/Dockerfile.ubi8

```diff
@@ -33,10 +37,29 @@ COPY --chown=appuser:appuser include/etc/confluent/docker /etc/confluent/docker
 RUN mkdir -p /etc/${COMPONENT} /etc/${COMPONENT}/secrets /var/log/${COMPONENT} /usr/logs \
   && chown appuser:appuser -R /etc/${COMPONENT} /var/log/${COMPONENT} /usr/logs
 
-RUN echo "===> Installing confluent-hub..." \
-  && wget http://client.hub.confluent.io/confluent-hub-client-latest.tar.gz \
-  && tar xf confluent-hub-client-latest.tar.gz \
-  && rm confluent-hub-client-latest.tar.gz
+RUN yum -q -y update \
+    && echo "===> Adding confluent repository...${CONFLUENT_PACKAGES_REPO}" \
+    && rpm --import ${CONFLUENT_PACKAGES_REPO}/archive.key \
+    && printf "[Confluent.dist] \n\
+name=Confluent repository (dist) \n\
+baseurl=${CONFLUENT_PACKAGES_REPO}/7 \n\
```

Comment:
Minor: trick I learned from Andrew - you can use `/$releasever` instead of `/7` - this way even if someone changes the base image in the future to be based on redhat/centos 8, this code won't have to change.

Reply:
I'm going to blow your mind here: `ubi8` *is* a RHEL8 based container - The "7" here was a work around in 5.5.x since EL7 binaries should be ABI compatible with RHEL8's platform, but *now* that we *are* building RHEL8 packages in 6.0.x and beyond, this change *is* advisable.

Reply:
need to follow up on this.  We only added /8 recently (we have used UBI8, but with /7 anyways so $releasever may actually have been wrong). In 6.0.x this maybe can be changed across images.